docs/manuals/source/DeveloperGuide/netprotocol.rst

```diff
@@ -245,3 +250,23 @@ The following diagram shows the activity on a Bareos TCP server (i.e. a |dir| or
   :Handle client message;
   end
 
+.. _DeveloperGuideNetworkSequenceDiagrams:
+
+Network Sequence Diagrams
+-------------------------
+
+This chapter contains diagrams of the network traffic for the most common jobs. The jobs are simplified as much as possible to show the essence of the network traffic. Refer to chapter :ref:`DeveloperGuideJobExecution` for sequence diagrams with the internal program flow.
+
+Backup of a Single File
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Comprehensive Diagram
+^^^^^^^^^^^^^^^^^^^^^
+
+.. include:: netprotocol/backup_of_one_file_comprehensive.puml
```

Comment:
consider using ..uml instead and move the caption from your puml to rst document